packages/backend/src/constants.ts

```diff
@@ -17,5 +17,6 @@
  ***********************************************************************/
 
 // Image related
-export const bootcImageBuilderContainerName = '-bootc-image-builder';
-export const bootcImageBuilderName = 'quay.io/centos-bootc/bootc-image-builder:latest-1714633180';
+export const bootcImageBuilder = 'bootc-image-builder';
+export const bootcImageBuilderCentos = 'quay.io/centos-bootc/bootc-image-builder:latest-1714474808';
```

Comment:
you changed the image to `latest-1714474808` which is untested

Reply:
Doh, forgot I was testing that on the side. Fixed.